runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/CreateWorkflowResponse.java

```diff
@@ -6,29 +6,29 @@
 import java.nio.ByteBuffer;
 import java.util.UUID;
 
-import static org.corfudb.format.Types.OrchestratorResponseType.WORKFLOW_ID;
+import static org.corfudb.format.Types.OrchestratorResponseType.WORKFLOW_CREATE;
 
 /**
  * AddNodeResponse returns the UUID of the add node workflow that was requested.
  * @author Maithem
  */
-public class AddNodeResponse implements Response {
+public class CreateWorkflowResponse implements Response {
```

Comment:
Also... this name doesn't match anymore...

Reply:
done.